tools/postgres/ssl/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3.5'
+services:
+  db:
+    build: .
+    command: -c ssl=on -c ssl_cert_file=/var/lib/postgresql/postgres.server.crt -c ssl_key_file=/var/lib/postgresql/postgres.server.key -c ssl_ca_file=/var/lib/postgresql/rootCA.crt -c hba_file=/var/lib/postgresql/pg_hba.conf
+    restart: always
+    environment:
+      POSTGRES_USER: ${KUMA_STORE_POSTGRES_USER:-kuma}
+      POSTGRES_PASSWORD: ${KUMA_STORE_POSTGRES_PASSWORD:-kuma}
+      POSTGRES_DB: ${KUMA_STORE_POSTGRES_DB_NAME:-kuma}
+    ports:
+      - ${KUMA_STORE_POSTGRES_PORT:-15432}:5432
+    volumes:
+      - ../../../app/kumactl/cmd/install/testdata/postgres_schema.golden.sql:/docker-entrypoint-initdb.d/init.sql
+      - ./certs/rootCA.crt:/var/lib/postgresql/rootCA.crt:ro
+      - ./certs/postgres.server.crt:/var/lib/postgresql/postgres.server.crt:ro
```

Comment:
Why we are not mounting server's TLS key ? (`/var/lib/postgresql/postgres.server.key`)

Reply:
Unfortunately, I could not find a way to mount it with proper permissions

Reply:
Let's add a note about it next to this line, and mention how it is solved instead.